lib/link_thumbnailer/processor.rb

```diff
@@ -120,6 +137,10 @@ def valid_response_format?(response)
       false
     end
 
+    def too_big_download_size?(size)
+      download_size_limit && (size.to_i > download_size_limit)
```

Comment:
probably want to call `to_i` here as well to make sure it's an integer

Reply:
the other option is to set a high limit by default and just forget about the `nil` check here

Reply:
Setting a reasonably high limit as default seems like something no one will object to.